### PR TITLE
added port list for pasv connection

### PR DESCRIPTION
**Pull request by hexfati**

### CODE REVIEW

File: wadas/ui/configure_ftp_cameras_dialog.py [hunk 1]
```diff
@@ -268,6 +276,27 @@ def validate(self):
         else:
             self.ui.label_errorMessage.setText("No server port provided!")
             valid = False
+        start_passive_port = self.ui.lineEdit_passive_port_range_start.text()
+        end_passive_port = self.ui.lineEdit_passive_port_range_end.text()
+        if not start_passive_port:
+            self.ui.label_errorMessage.setText("Start passive port not provided!")
+            valid = False
+        elif start_passive_port and (int(start_passive_port) < 1 or int(start_passive_port) > 65535):
```
Comment (cdimauro) [inline wadas/ui/configure_ftp_cameras_dialog.py:284]: elif ((start_passive_port_number := int(start_passive_port)) < 1 or start_passive_port_number > 65535:

File: wadas/ui/configure_ftp_cameras_dialog.py [hunk 2]
```diff
@@ -268,6 +276,27 @@ def validate(self):
...
+            self.ui.label_errorMessage.setText("Invalid start passive port provided!")
+            valid = False
+        if not end_passive_port:
+            self.ui.label_errorMessage.setText("End passive port not provided!")
+            valid = False
+        if end_passive_port and (int(end_passive_port) < 1 or int(end_passive_port) > 65535):
```
Comment (cdimauro) [inline wadas/ui/configure_ftp_cameras_dialog.py:290]: if (end_passive_port_number := int(end_passive_port)) < 1 or end_passive_port_number > 65535:

File: wadas/ui/configure_ftp_cameras_dialog.py [hunk 3]
```diff
@@ -268,6 +276,27 @@ def validate(self):
...
+            self.ui.label_errorMessage.setText("Invalid passive end port provided!")
+            valid = False
+        if start_passive_port and end_passive_port:
+            if int(start_passive_port) > int(end_passive_port):
```
Comment (cdimauro) [inline wadas/ui/configure_ftp_cameras_dialog.py:294]: if start_passive_port_number > end_passive_port_number:

File: wadas/ui/configure_ftp_cameras_dialog.py [hunk 4]
```diff
@@ -268,6 +276,27 @@ def validate(self):
...
+                self.ui.label_errorMessage.setText("Start passive port cannot be greater than end passive port!")
+                valid = False
+            elif int(end_passive_port) < int(start_passive_port):
```
Comment (cdimauro) [inline wadas/ui/configure_ftp_cameras_dialog.py:297]: elif end_passive_port_number < start_passive_port_number:

File: wadas/ui/configure_ftp_cameras_dialog.py (hunk 1 above)
Comment (stefanodellosa-personal) [reply wadas/ui/configure_ftp_cameras_dialog.py:284]: if we remove **elif start_passive_port and ...** it will fail on empty string

Comment (cdimauro) [reply wadas/ui/configure_ftp_cameras_dialog.py:284]: The elif is executed only if start_passive_port is a string which is not empty.

**Pull request merged**